.github/workflows/CI.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  spindle-serial-ubuntu:
+    name: Testsuite (Serial, Ubuntu)
+    environment: Spindle CI
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Check out Spindle
+        uses: actions/checkout@v5
+        with:
+          path: 'Spindle'
```

Comment:
```suggestion
```

Did you all decide to clone into `Spindle/` for a particular use case? Normally we recommend to clone without a path which clone into the CI/CD runner current working directory. (And then minimizing the number of cd hierarchies you need later on.)